src/main/java/uk/gov/hmcts/ccd/domain/model/std/MessageInformation.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.ccd.domain.model.std;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@SuppressWarnings("checkstyle:SummaryJavadoc") // Javadoc predates checkstyle implementation in module
+public class MessageInformation {
+    @JsonIgnore
+    private Long id;
+    @JsonProperty("user_id")
+    private String userId;
+    @JsonProperty("jurisdiction_id")
+    private String jurisdictionId;
+    @JsonProperty("case_type_id")
+    private String caseTypeId;
+    @JsonProperty("case_id")
+    private String caseId;
+    @JsonProperty("event_timestamp")
+    private LocalDateTime eventTimestamp;
+    @JsonProperty("event_instance_id")
+    private String eventInstanceId;
```

Comment:
Could make this a `Long` to avoid needing to convert between `String`/`Long` whenever creating the object. 
```suggestion
    private Long eventInstanceId;
```

Reply:
done 👍